webpack.js

```diff
@@ -15,6 +15,8 @@ webpackConfig.stats = {
 
 const appId = 'assistant'
 webpackConfig.entry = {
+	generationPage: { import: path.join(__dirname, 'src', 'imageGenerationPage.js'), filename: appId + '-imageGenerationPage.js' },
+	reference: { import: path.join(__dirname, 'src', 'reference.js'), filename: appId + '-reference.js' },
```

Comment:
As we'll add other reference stuff and they will probably have their own script each, let's name this one `imageGenerationReference.js`